src/libraries/Common/src/Interop/Browser/Interop.JavaScript.Array.cs

```diff
@@ -0,0 +1,40 @@
+using System;
```

Comment:
Could you add interop library as well as we can add tests there?

Reply:
Marek could you explain a little more.  Not sure I understand what you are wanting.

Reply:
It's not clear if the code actually compiles right now and I think we should have good unit test coverage for the interop code. I'd suggest we create System.Runtime.Interop.JavaScript library which won't be public but will include all Interop files and we also add /test under that project as a place for interop unit tests.

/cc @lewing

Reply:
Got it.  I did not see a library for the other platforms like that so did not think it was needed.

Reply:
Added the initial `sln` and `csproj` file for `System.Runtime.Interop.JavaScript` library

Reply:
Should this go in the same folder as interop code?

Reply:
Addressed in recent commits.  Move to /src/libraries